skins/oasis/modules/templates/Oasis_Index.php

```diff
@@ -95,9 +95,14 @@
 	<!-- Combined JS files and head scripts -->
 	<?= $jsFiles ?>
 <? endif ?>
+UserLoggedIn: <?= $isUserLoggedIn ?>
+CookieSyncEnabled: <?= $cookieSyncEnabled ?>
 <? if ( $isUserLoggedIn && $cookieSyncEnabled ): ?>
 	<?= F::app()->renderView( 'AutoLoginService', 'Index' ) ?>
 <? endif ?>
+<? if ( !$isUserLoggedIn && $cookieSyncEnabled ): ?>
+	<?= F::app()->renderView( 'AutoLoginService', 'Passive' ) ?>
```

Comment:
Shouldn't this be rendered only when on fandom.com?

Reply:
Well that should be enough in my opinion but the ticket stands: `If you are logged in on wikia.com, authentication cookies are synced when visiting a fandom.com wiki (and vice-versa)`